packages/block-library/src/cover/block.json

```
@@ -139,6 +138,14 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"filter": {
+			"duotone": true
+		}
+	},
+	"selectors": {
+		"filter": {
+			"duotone": "> .wp-block-cover__image-background, > .wp-block-cover__video-background"
```

```suggestion
			"duotone": ".wp-block-cover > .wp-block-cover__image-background, .wp-block-cover > .wp-block-cover__video-background"
```

I think this is the correct selector, otherwise the duotone filter won't be applied via the global styles.

---

You are right. I checked locally, and after updating the selector, the duTone global style is working fine for the cover block. Thanks for pointing this out. I have updated the changes on the next commit.